README.md

```diff
@@ -19,26 +19,17 @@ unicode-case-mapping
 <br>
 
 Fast mapping of `char` to lowercase, uppercase, or titlecase in Rust using
-Unicode 12.1 data.
+Unicode 13.0 data.
 
 Usage
 -----
 
-`Cargo.toml`:
-
-```toml
-[dependencies]
-unicode-case-mapping = "0.1.0"
-```
-
```

Comment:
this isn't necessary or desirable any more?

Reply:
I removed that from the README of all of them since I think it's reasonable to assume people know how to add the crate to their project, and for us it creates busy work of needing to keep the version in the README synced with the published version.

Reply:
Makes sense!